src/frontend/src/lib/components/ui/ButtonControl.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	export let ariaLabel: string;
+	export let styleClass: string | undefined = '';
```

Comment:
It's a bit weird to read I have to say: `styleClass` can be undefined but is never undefined 🤪. One consumer can set `styleClass={undefined}` and it's still renders "undefined" in the DOM.

I suggest to fallback to `""` within the class assignation.

Reply:
Ok 👍